sdk/identity/identity/tests.yml

```diff
@@ -3,6 +3,22 @@ trigger: none
 extends:
     template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
     parameters:
+      PreSteps:
+      - task: AzureCLI@2
```

Comment:
should this live under eng/ somewhere so that other packages could use it? Or do you plan to refactor later?

Reply:
I was planning to talk to EngSys about making this is a supported scenario - and part of that is making it available via config or something. I'll also need to use this for Managed HSM deployments but I figured let's start locally and ensure it's stable first - does that seem reasonable?

Reply:
@maorleger @jeremymeng We initially considered putting this in eng/common, but decided that since it's only used by identity we would keep it at the top level. Given that MHSM also needs it, I will discuss with @weshaggard whether we want to move it into eng/common now.

Reply:
SGTM, thank you! FWIW [this](https://teams.microsoft.com/l/message/19:59dbfadafb5e41c4890e2cd3d74cc7ba@thread.skype/1723593132403?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1723593132403&teamName=Azure%20SDK&channelName=Engineering%20System%20%F0%9F%9B%A0%EF%B8%8F&createdTime=1723593132403) is the scenario I am thinking I can use this for in keyvault - while this uses a PowerShell cmdlet my understanding of the problem is that the original federated token expires before we get a chance to call this (token expiry is 10m, MHSM can often take longer to deploy).

Hope that sets the context for keyvault at least, it's also possible that there's a better way hence I am tying the two conversations here 😄